src/NewPost.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from "react";
+import { data } from "./data";
+
+export class NewPost extends Component {
+  state = {
+    title: "",
+    author: "",
+    createdOn: "",
+    body: "",
+    tags: ""
+  };
+
+  createDateOnSubmit = () => {
+    const date = new Date();
+    let year = date.getFullYear();
+    let month = date.getMonth();
+    let day = date.getDate();
+    const dateNow = `${year}-${month + 1}-${day}`;
+    return dateNow;
+  };
+
+  createUniqueIdOnSubmit = () => {
+    const id = data.length;
+    return id;
+  };
+
+  convertTagsToArray = tags => {
+    const splitOn = ",";
+    let resultArray = tags.split(splitOn);
```

Comment:
can chain the `split` and `map` so that you can use a `const` here. you can even just `return tags.split(...).map(...)` directly since you're not using `resultArray` for anything else before returning it